### DISCUSSION TITLE
Plugin for rating multiple audios on a slider

### DISCUSSION THREAD

**Discussion by Semeriuss** [jspsych/jsPsych#3191] [Ideas]
I was trying to develop a [MUSHRA](https://forcetechnology.com/-/media/force-technology-media/pdf-files/unnumbered/senselab/product-sheet-itu-r-bs,-d-,1534-3-mushra.pdf) test on jsPsych but couldn't find a suitable plugin for it. I checked this [discussion ](https://github.com/jspsych/jsPsych/discussions/2892) and followed up on this [discussion](https://github.com/jspsych/jsPsych/discussions/1147) to develop a new plugin suited for MUSHRA-style (multiple audio slider rating) tests. 

Here's the current state of the plugin. The yellow button (for the reference) is optional and the audio position tracker syncs smoothly between audios (without necessarily having to start from the beginning for each audio). The written labels on top can also be modified/excluded. Each label gets highlighted depending on the position of the rating slider. 

![training](https://github.com/jspsych/jsPsych/assets/50263682/e60e4af7-6944-4f95-adcb-efa7d3813c18)

Let me know if this would this be helpful to anyone or if you want more information about what responses are collected, etc.

**Comment by jodeleeuw** [comment]
@Semeriuss this looks great!

A contribution to https://github.com/jspsych/jspsych-contrib would be very welcome.
